### ISSUE TITLE
Unable to resolve module 'stream'

### ISSUE THREAD

**Issue by Jahans3** [aws/aws-amplify#141]
There appears to be a node core module imported from this package via the `aws-amplify` package.

```
error: bundling failed: Error: Unable to resolve module `stream` from `/Users/joshua/Documents/side-projects/RoomShakeClient/node_modules/cipher-base/index.js`: Module does not exist in the module map
```

Does anybody know a fix for this? I've downgraded a few versions but it doesn't seem to help

**Comment by 7772** [comment]
Hi @Jahans3 .
I solved this error. #140

**Comment by Jahans3** [comment]
Thanks @7772 !

**Comment by Jahans3** [comment]
Reopening as unfortunately workaround did not work

**Comment by mlabieniec** [comment]
@Jahans3 this should be resolved in v0.1.34 (just published today). Please re-open if you are still seeing the issue.

**Comment by Jahans3** [comment]
@mlabieniec could you apply the fix to aws-amplify-react-native too please
